charters/DataOperations/charter.md

```diff
@@ -0,0 +1,89 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team is responsible for providing the scientific community with effective tools and resources which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution.
+
+## Definitions [Optional]
+
+Data submission: a set of raw data derived from the same experimental process, and the metadata describing that process
+Data release: the point at which metadata and data from a single data submission become publicly available
+Data distribution: the packaging of metadata and data from multiple data submissions in meaningful ways and any announcements or documentation supporting it
+
+## Objectives
+
+The objective of the Data Operations is to provide the scientific community with effective tools and resources from the DCP which maximize the value and use of HCA data
+
+## In-scope
+
+- Data operations
+  - Engage with all DCP components to identify deliverables, including data, pipelines, metadata, standards, and user features, which maximize the value and consistency of HCA data
```

Comment:
Raised these to top line objective in 0de0c59. @jonahcool let me know if this addresses your concern. If not, we can work to do so during oversight review or iterate before then - whichever you prefer.

Reply:
I am not sure I understand this. How does it relate to the longterm roadmap process? Our deliverables will be founded in the longterm road map and manifest as translations of the longterm road maps into implementation details in the engineering plan. Does this mean you are engaging the teams (and users) to identify things that should be distributed to users in the context of releases? If so, I like that a lot but this would be an example where giving a larger context of distribution, curation, and holistic packaging will help.

Reply:
@TimothyTickle you are correct that a large portion of this was missing (taking those deliverables to the community). Added an objective in 79b631f that hopefully clarifies this.

Reply:
Resolving this but please reopen if there's still concerns/questions